src/content/en/tools/chrome-devtools/_toc.yaml

```diff
@@ -41,7 +41,7 @@ toc:
         path: /web/tools/chrome-devtools/device-mode/testing-other-browsers
     - title: Remote Debugging Android Devices
       section:
-      - title: Overview
+      - title: Get Started
```

Comment:
Change this back to overview and the in-page title back to 'Remote Debugging Android Devices'. This is consistent with the rest of the site.